### DISCUSSION TITLE
Desktop icon set

### DISCUSSION THREAD

**Discussion by cdaaawg** [oxen-io/session-desktop#2711] [General]
Hi all!  I am using Session via the .appimage file downloaded from the Session website and would like to create a desktop application launcher. Is there a downloadable "official" icon suitable for attaching to a Linux desktop application launcher? Thanks for any enlightenment.

**Comment by securegh** [comment]
The icons are is inside the appimage, need to be extracted https://superuser.com/questions/1301583/how-can-i-extract-files-from-an-appimage
I used this tool to manage and create appimage launcher appimagepool 
https://github.com/prateekmedia/appimagepool
